pkg/security/secl/rules/ruleset.go

```diff
@@ -148,8 +148,8 @@ func (rs *RuleSet) AddRules(parsingContext *ast.ParsingContext, pRules []*Policy
 	return result
 }
 
-// PopulateFieldsWithRuleActionsData populates the fields with the data from the rule actions
-func (rs *RuleSet) PopulateFieldsWithRuleActionsData(policyRules []*PolicyRule, opts PolicyLoaderOpts) *multierror.Error {
+// populateFieldsWithRuleActionsData populates the fields with the data from the rule actions
+func (rs *RuleSet) populateFieldsWithRuleActionsData(policyRules []*PolicyRule, opts PolicyLoaderOpts) *multierror.Error {
```

Comment:
`populateFieldsWithRuleActionsData` was made public by https://github.com/DataDog/datadog-agent/pull/28708 to fix the backend validation of rules, so I think we should keep it public for now.

Reply:
Will revert this change

Reply:
done